light-client/src/predicates.rs

```diff
@@ -95,7 +95,7 @@ pub trait VerificationPredicates: Send {
         Ok(())
     }
 
-    /// Check that the given header is within the trusting period, adjusting for clock drift.
+    /// Check that the trusted header is within the trusting period, adjusting for clock drift.
     fn is_within_trust_period(
         &self,
         header: &Header,
```

Comment:
Nit: maybe we can make it more explicit that we are expecting a trusted header by renaming the param to: `trusted_header`?

Reply:
I think that's a great idea